### ISSUE TITLE
Can't build program unless bc is installed

### ISSUE THREAD

**Issue by thsowers** [raphaelsc/Am-I-affected-by-Meltdown#14]
Worked fine yesterday, pulled the latest code and ran into this issue

```
/bin/sh: 1: bc: not found
g++ -I. --std=c++11 -O0 --no-pie -mrtm -fPIC -c -o meltdown_checker.o meltdown_checker.cc 
In file included from meltdown_checker.cc:49:0:
assembly_utils.hh:36:2: warning: #warning "Using native impl. of TSX due to GCC version older than 4.8. No need to worry about it!" [-Wcpp]
 #warning "Using native impl. of TSX due to GCC version older than 4.8. No need to worry about it!"
  ^
In file included from /usr/lib/gcc/x86_64-linux-gnu/5/include/immintrin.h:83:0,
                 from /usr/lib/gcc/x86_64-linux-gnu/5/include/x86intrin.h:46,
                 from /usr/include/x86_64-linux-gnu/c++/5/bits/opt_random.h:33,
                 from /usr/include/c++/5/random:50,
                 from /usr/include/c++/5/bits/stl_algo.h:66,
                 from /usr/include/c++/5/algorithm:62,
                 from meltdown_checker.cc:41:
assembly_utils.hh:38:22: error: expected class-name before numeric constant
 static constexpr int _XBEGIN_STARTED = ~0u;
                      ^
assembly_utils.hh:38:22: error: expected ‘)’ before numeric constant
In file included from meltdown_checker.cc:49:0:
assembly_utils.hh: In function ‘int _xbegin()’:
assembly_utils.hh:41:24: error: ambiguating new declaration of ‘int _xbegin()’
 inline int _xbegin(void) {
                        ^
In file included from /usr/lib/gcc/x86_64-linux-gnu/5/include/immintrin.h:83:0,
                 from /usr/lib/gcc/x86_64-linux-gnu/5/include/x86intrin.h:46,
                 from /usr/include/x86_64-linux-gnu/c++/5/bits/opt_random.h:33,
                 from /usr/include/c++/5/random:50,
                 from /usr/include/c++/5/bits/stl_algo.h:66,
                 from /usr/include/c++/5/algorithm:62,
                 from meltdown_checker.cc:41:
/usr/lib/gcc/x86_64-linux-gnu/5/include/rtmintrin.h:50:1: note: old declaration ‘unsigned int _xbegin()’
 _xbegin (void)
 ^
In file included from meltdown_checker.cc:49:0:
assembly_utils.hh: In function ‘void _xend()’:
assembly_utils.hh:48:13: error: ‘void _xend()’ redeclared inline without ‘gnu_inline’ attribute
 inline void _xend(void) {
             ^
In file included from /usr/lib/gcc/x86_64-linux-gnu/5/include/immintrin.h:83:0,
                 from /usr/lib/gcc/x86_64-linux-gnu/5/include/x86intrin.h:46,
                 from /usr/include/x86_64-linux-gnu/c++/5/bits/opt_random.h:33,
                 from /usr/include/c++/5/random:50,
                 from /usr/include/c++/5/bits/stl_algo.h:66,
                 from /usr/include/c++/5/algorithm:62,
                 from meltdown_checker.cc:41:
/usr/lib/gcc/x86_64-linux-gnu/5/include/rtmintrin.h:61:1: note: ‘void _xend()’ previously defined here
 _xend (void)
 ^
Makefile:12: recipe for target 'meltdown_checker.o' failed
make: *** [meltdown_checker.o] Error 1
```

Fixed by installing `bc`

**Comment by raphaelsc** [comment]
thanks for the report, i'll remove the dependency on bc

**Comment by raphaelsc** [comment]
fixed after 923165ecff690963cbf5bf53b741ec40dd8037f2
